### PR TITLE
chore(website): Fix broken links, re-add Heroku docs

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Follow up https://github.com/cloudquery/cloudquery/pull/5104. Re-adds the Heroku docs, fixes broken links

### CODE REVIEW

File: website/components/mdx/_configure.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Callout } from 'nextra-theme-docs'
 
 ## Configure Destination Plugin
 
-Destination plugins define where you will be syncing your data to (See: [available destinations](/docs/plugins/destinations)).
+Destination plugins define where you will be syncing your data to (See: [available destinations](/docs/plugins/destinations/overview)).
```
Comment (erezrokah) [inline website/components/mdx/_configure.mdx:5]: This is technically not needed as we have a redirect, but saves users the extra hop

File: website/pages/blog/announcing-cloudquery-snowflake-destination.md [hunk 2]
```diff
@@ -13,21 +13,21 @@ import { BlogHeader } from "../../components/BlogHeader"
...
+Snowflake is already used as a ["Security data lake"](https://www.snowflake.com/en/data-cloud/workloads/cybersecurity/). CloudQuery brings all your cloud [infrastructure configuration data](/docs/plugins/sources/overview) to the same place, consolidating your security and infrastructure data in one place and enabling new insights.
 
 ### Historical data
 
 Maintaining historical data is a common use case for data warehouses. By using Snowflake as a destination, you can now store all your cloud infrastructure data in a data warehouse for long term storage, analysis and investigation use-cases.
 
 ## Syncing data
 
-Syncing data to snowflake can be done as with any other destination plugin, so check out our [Quickstart](../docs/quickstart) and [Snowflake recipe](../docs/recipes/destinations/snowflake).
+Syncing data to snowflake can be done as with any other destination plugin, so check out our [Quickstart](/docs/quickstart) and [Snowflake recipe](/docs/recipes/destinations/snowflake).
```
Comment (erezrokah) [inline website/pages/blog/announcing-cloudquery-snowflake-destination.md:30]: The issue with relative links is that if we move the file the link breaks and it's also hard (at least for me) to calculate the result of the relative path.

File: website/pages/blog/aws-kms-key-grants-deep-dive.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ author: jsonkao
 import { BlogHeader } from "../../components/BlogHeader"
 
 <BlogHeader/>
-We recently published a [blog post](https://www.cloudquery.io/blog/aws-encryption-and-multi-account-access) about Encryption in AWS and Multi-Account Access.  As a follow-up to that post, we’re focusing on encryption access in AWS and information on [how CloudQuery can help](https://www.cloudquery.io/docs/plugins/sources) including how to audit access to those encryption keys and the underlying data to help assist with your security and compliance needs.
```
Comment (erezrokah) [inline website/pages/blog/aws-kms-key-grants-deep-dive.md:13]: This change ensures that the correct host is used in deploy previews

File: website/pages/docs/plugins/sources/cloudflare/overview.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ In order to fetch information from Cloudflare, `cloudquery` needs to be authenti
 
 - Export the `CLOUDFLARE_API_TOKEN` environment variable before running `cloudquery sync`.
 - Export the `CLOUDFLARE_EMAIL` and `CLOUDFLARE_API_KEY` environment variables before running cloudquery
-- Specifying either the `api_token` or `api_email, api_key` parameters in the YAML configuration (See [Configuration](./docs/configuration.md) for more details).
```
Comment (erezrokah) [inline website/pages/docs/plugins/sources/cloudflare/overview.md:11]: This link was broken

File: website/pages/docs/plugins/sources/overview.mdx [hunk 5]
```diff
@@ -31,9 +31,9 @@ Official source plugins follow [release stages](source-plugin-release-stages).
 |--------------------------------------------------------------|
 | [Yandex](https://github.com/yandex-cloud/cq-provider-yandex) |
 
-# Source Plugin Release Stages
+## Source Plugin Release Stages
 
-[Official source plugins](sources) go through two release stages: Preview, and GA.
```
Comment (erezrokah) [inline website/pages/docs/plugins/sources/overview.mdx:36]: Similar to release stages, `sources` used to be a separate page. The fix is to link to the anchor

File: website/pages/docs/reference/source-spec.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
 # Source Spec Reference
 
-Following are available options for the top level source plugin `spec` object. 
+Following are available options for the top level source plugin `spec` object.
 
-Note: For configuring individual plugins, please refer to the configuration section from the relevant plugins from [here](https://www.cloudquery.io/docs/plugins/sources). (e.g. [AWS plugin configuration](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/docs/configuration.md)). Also check out our [recipes](https://www.cloudquery.io/docs/recipes/overview) for common configuration examples.
```
Comment (erezrokah) [inline website/pages/docs/reference/source-spec.md:5]: This links to 
image

so I changed it to link directly to the AWS doc on the Website

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**